src/modules/Exposure/info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Exposure",
+  "description": "Change the exposure of the image by given exposure value",
+  "inputs": {
+      "exposure": {
+          "type": "range",
+          "desc": "exposure value for the new image",
+          "default": "0",
```

Comment:
Why is the default 0. Why not make it something more representative?

Reply:
Also make all the number strings actual numbers.

Reply:
Representative as in? I chose `0` as the default to represent the normal image. The range can be used to increase or decrease exposure, I think that's fine.

Reply:
I mean like the change should be visible. Zero wouldn't make any change to the img right?

Reply:
Earlier rotate module was at zero but @jywarren suggested to change it to 90 and also if this module is used in cli, defaulting to something other than 0 would be better i guess

Reply:
I'll make it to 1 then